perf/src/index.tsx

```diff
@@ -61,9 +62,15 @@ const renderCycle = async (
   return profilerMeasure
 }
 
-const satisfiesFilter = (componentName: string, filter: string) => {
-  return componentName.toLowerCase().startsWith(filter.toLowerCase())
-}
+const satisfiesFilter = (componentName: string, filter: string) =>
+  minimatch(componentName, filter || '*', {
+    matchBase: true,
+  })
+
+const sortObjectKeys = o =>
```

Comment:
I will be grad to replace this with Lodash equivalent, if you know it, please propose 😺

Reply:
I've always thought order of object keys is not guaranteed 🤔

Reply:
> Current Language Spec (since ES2015) insertion order is preserved, except in the case of keys that parse as integers (eg "7" or "99"), where behavior varies between browsers. For example, Chrome/V8 does not respect insertion order when the keys are parse as numeric.

https://stackoverflow.com/a/23202095/6488546  ¯\_(ツ)_/¯